README.md

```diff
@@ -1,10 +1,72 @@
 # Spring Boot application template
 
+[![Build Status](https://travis-ci.org/hmcts/spring-boot-template.svg?branch=master)](https://travis-ci.org/hmcts/spring-boot-template)
+
 ## Purpose
 
 The purpose of this template is to speed up the creation of new Spring applications within HMCTS
 and help keep the same standards across multpile teams. If you need to create a new app, you can
 simply use this one as a starting point and build on top of it.
 
+TODO: complete the readme
```

Comment:
Is this TODO still needed?

Reply:
Yes, I'm going to add more information to the readme as a part of current story, so that people know how to use this template.